Makefile

```diff
@@ -0,0 +1,25 @@
+HASGOCILINT := $(shell which golangci-lint 2> /dev/null)
+
+
+ifdef HASGOCILINT
+    GOLINT=golangci-lint
+else
+    GOLINT=bin/golangci-lint
+endif
+
+# Dependency versions
+GOLANGCI_VERSION = 1.53.3
```

Comment:
Looks like we don't need these since we are not using the golangci-lint yet. Do you plan adding this later?